SEL32/sel32_con.c

```diff
@@ -669,7 +581,10 @@ t_stat con_srvi(UNIT *uptr) {
                 sim_debug(DEBUG_CMD, dptr,
                 "con_srvi readch unit %02x: CMD %08x read %02x u4 %02x incnt %02x\n",
                 unit, uptr->CMD, ch, uptr->u4, con_data[unit].incnt);
-            sim_activate(uptr, 30);         /* do this again */
+//JB        sim_clock_coschedule(uptr, 1000);
+//JB        sim_activate(uptr, 30);         /* do this again */
+//??        sim_activate(uptr, 300);        /* do this again */
+            sim_activate(uptr, 500);        /* do this again */

```

Comment:
Could you clean up all this commented out code everywhere?  It's hard to make sense out of any of them, and this spot with a half dozen nearly identical lines is particularly confusing.

Reply:
Information to myself about various values I have tried during the past.  I try different timing for diags, UTX(UNIX), and MPX and they help me to remember what I tried.

Reply:
The source control system has history; it isn't necessary to keep it right in the source file active code.